client-contract/src/DealClient.sol

```diff
@@ -312,7 +313,7 @@ contract DealClient {
     function uintToBigInt(
```

Comment:
We should get rid of these 2 functions altogether and use utils in the new version of the zondax libs here: https://github.com/Zondax/filecoin-solidity/blob/master/contracts/v0.8/utils/BigInts.sol
since this is being updated anyway